protocols/imap/src/main/java/org/apache/james/imap/processor/base/AbstractChainedProcessor.java

```diff
@@ -54,6 +56,7 @@ public void process(ImapMessage message, Responder responder, ImapSession sessio
             M acceptableMessage = (M) message;
             try (Closeable closeable = addContextToMDC(acceptableMessage)) {
                 try {
+                    LOGGER.debug("Processing {}", message);
```

Comment:
implicit toString?

Reply:
@mbaechler claimed the opposite. Link https://github.com/linagora/james-project/pull/3402#discussion_r432375436

Honnestly, I don't care, both are ok to me. I'll keep the current version unless some strong divergences are expressed.